lib/file_transfer_agent/gcs_util.js

```diff
@@ -57,7 +70,15 @@ function GCSUtil(httpclient, filestream) {
   this.createClient = function (stageInfo) {
     const stageCredentials = stageInfo['creds'];
     const gcsToken = stageCredentials['GCS_ACCESS_TOKEN'];
-
+    //TODO: SNOW-1789759 the value is hardcoded now, but it should be server driven
+    const isRegionalUrlEnabled = (stageInfo.region).toLowerCase() === 'me-central2' || stageInfo.useRegionalUrl;
```

Comment:
this change should be already included in the master branch and released, I don't think it should be part of this PR

Reply:
Previously, this implementation was within the if statement that checks for the existence of a GCS token. I moved it to the beginning of the method because the endpoint value is needed to initialize the Axios instance.